CONTRIBUTING.md

```diff
@@ -303,6 +303,15 @@ You can determine the correct `sha256` like this:
 
 If the commit you need in `formal-ledger-specifications` is not on master, open a PR for your branch in the `formal-ledger-specifications` repository. This will create a branch with the updated generated code, which you can then use as described above. You will not be able to merge in `cardano-leder` master a reference to a commit not yet merged in `formal-ledger-specifications`.
 
+### To run conformance testing against a local build of the Agda ledger spec
```

Comment:
I can't tell what the difference is between this and the step above  "If the commit you need in `formal-ledger-specifications` is not on master"  - are they the same  thing?

Reply:
The step above refers to using a git remote that holds the Agda-generated Haskell code, in contrast, this step is about using a local folder that contains the code instead.

Reply:
Oh I see, so if I have formal-ledger-specifications repo locally and I had built it. Makes sense, thanks!

Reply:
That's actually reminds of an important point. It is necessary to add here a step that either links to or describes how to generate MAlonzo code locally